src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -238,6 +263,31 @@
     private static final String DELIVERYMEN_JOB = "com.minecolonies.coremod.job.Deliveryman";
 
     /**
+     * The Miners job description string.
+     */
+    private static final String MINER_JOB = "com.minecolonies.coremod.job.Miner";
+
+    /**
+     * The Fishermans job description string.
```

Comment:
Okay, so should just be a simple copy and paste? (I'm guessing I'd have to change the private to public?)